game/src/contrib/components/InteractionComponent.java

```diff
@@ -5,55 +5,79 @@
 
 import java.util.function.Consumer;
 
-public class InteractionComponent extends Component {
-    public static final int DEFAULT_RADIUS = 5;
+/**
+ * Allows interaction with the associated entity.
+ *
+ * <p>An interaction can be triggered by the player character with a button press when in range.
```

Comment:
Ein paar Zeilen weiter unten schreibst Du "`The interaction can be triggered using {@link #triggerInteraction()}.`" - das ist etwas widersprüchlich?

Ich würde zumindest die konkrete Beschreibung mit triggerInteraction hier hochziehen und den Satz mit dem Benutzer umformulieren: Der Benutzer kann einen Button drücken, was von System XYZ ausgewertet wird und welches dann die Methode triggerInteraction aufruft.

Reply:
hab den Absatz nach vorne geschoben und erklärt, dass das PlayerSystem beim drücken der Interaktion Taste auf der Tastatur eine Interaktion auslöst